src/ui/map/Map.svelte

```diff
@@ -59,9 +59,11 @@
       }, debounceTimeout);
     };
   }
-
   const debouncedMapZoomBBoxStore = getDebouncedMapZoomBBoxStore();
 
+  /* Custom bounding box */
+  const boundingBox = [-10.5, 47.5, 5.9, 57.8];
```

Comment:
Is the custom boundingBox something that should be in the config file ?

Reply:
Yeah, I did originally think that but wasn't sure if I should edit it, if you are okay with it I can pop it in there?